keel-web/src/main/kotlin/com/netflix/spinnaker/config/DefaultConfiguration.kt

```diff
@@ -33,10 +35,39 @@ import org.springframework.context.annotation.Primary
 import org.springframework.core.Ordered.HIGHEST_PRECEDENCE
 import org.springframework.scheduling.TaskScheduler
 import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
+
+private const val IPC_SERVER_METRIC = "controller.invocations"
 
 @EnableFiatAutoConfig
 @Configuration
-class DefaultConfiguration {
+class DefaultConfiguration(
+  val spectatorRegistry: Registry
+) : WebMvcConfigurer {
+
+  /**
+   * Enable controller metrics
+   */
+  override fun addInterceptors(interceptorRegistry: InterceptorRegistry) {
+
+    /**
+     * Path variables that will be added as tags to the metrics
+     *
+     * Lorin chose these particular variables arbitrary as potentially useful. Once we start consuming
```

Comment:
I like to put my rationale in code comments and identify myself, especially when I don't have a particularly good reason for a decision.